ledger_app.toml

```diff
@@ -6,6 +6,7 @@ devices = ["nanos", "nanox", "nanos+", "stax", "flex"]
 [use_cases]
 use_test_keys = "TEST_PUBLIC_KEY=1"
 dbg_use_test_keys = "DEBUG=1 TEST_PUBLIC_KEY=1"
+testing_dbg_use_test_keys = "DEBUG=1 TESTING=1 TEST_PUBLIC_KEY=1"
```

Comment:
not sure to understand the need of this added line

Reply:
It was useful for some swap e2e tests done locally with LL but I can remove it